runtime/interpreter/errors.go

```diff
@@ -349,6 +349,34 @@ func (e ArrayIndexOutOfBoundsError) Error() string {
 	)
 }
 
+// ArraySliceIndicesError
+//
+type ArraySliceIndicesError struct {
+	FromIndex int
+	UpToIndex int
+	Size      int
+	LocationRange
+}
+
+func (e ArraySliceIndicesError) Error() string {
+	return fmt.Sprintf(
+		"slice indices [%d:%d] are out of bounds (size %d)",
+		e.FromIndex, e.UpToIndex, e.Size,
+	)
+}
+
+// InvalidSliceIndexError is returned when a slice index is invalid, such as fromIndex > upToIndex
```

Comment:
This answers my question above about invalid slice ranges, but this should be included in the documentation.

Reply:
Good idea to add it to the documentation! Added it there and to the docstrings in ce4487eb2e16b444667ea9d654407ff3de22b376